ocaml/xapi/xapi_globs.ml

```diff
@@ -1007,6 +1009,9 @@ let other_options = [
 
   "gpumon_stop_timeout", Arg.Set_float gpumon_stop_timeout,
     (fun () -> string_of_float !gpumon_stop_timeout), "Time to wait after attempting to stop gpumon when launching a vGPU-enabled VM.";
+
+  "reboot_required_hfxs", Arg.Set_string reboot_required_hfxs,
+    (fun () -> !reboot_required_hfxs), "File to query hotfix uuids which require reboot";
```

Comment:
This looks good now, but please also add an entry to scripts/xapi.conf (as an example, commented out).